rules/false_positives/clickhouse.yara

```diff
@@ -0,0 +1,15 @@
+rule clickhouse_binary : override {
+  meta:
+    malware_PlugX_config = "high"
+    description = "clickhouse"
+    original_severity = "critical"
+  strings:
+    $clickhouse_binary = "/usr/bin/clickhouse"
+    $clickhouse_client = "clickhouse-client"
+    $clickhouse_service = /clickhouse-\w{0,32}/ fullword
+    $clickhouse_server = "clickhouse-server"
+    $clickhouse_site = "https://clickhouse.com"
+    $usage = "Usage: ./clickhouse"
+  condition:
+    all of them
```

Comment:
That seems fragile - how about: `75% of them`

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).